searching/quick_select.c

```diff
@@ -0,0 +1,64 @@
+#include <stdio.h>
+// function to swap variable
+void swap(int* a, int* b)
```

Comment:
Functions should be documented explaining the use of its parameters, a brief description, and what it returns.